code/controllers/master.dm

```diff
@@ -195,7 +195,7 @@ ADMIN_VERB(cmd_controller_view_ui, R_SERVER|R_DEBUG, "Controller Overview", "Vie
 			"cost_ms" = subsystem.cost,
 			"tick_usage" = subsystem.tick_usage,
 			"usage_per_tick" = average,
-			"tick_overrun" = subsystem.tick_overrun,
```

Comment:
~~shouldn't we rename the var too~~

Reply:
~~technically the var name is correct because it does overrun the allocated subsystem tick time~~